supercommand.go

```diff
@@ -335,13 +348,16 @@ func (c *SuperCommand) SetCommonFlags(f *gnuflag.FlagSet) {
 	if c.Log != nil {
 		c.Log.AddFlags(f)
 	}
+	if c.globalFlags != nil {
+		c.globalFlags.AddFlags(f)
+	}
 	f.BoolVar(&c.showHelp, "h", false, helpPurpose)
 	f.BoolVar(&c.showHelp, "help", false, "")
 	// In the case where we are providing the basis for a plugin,
 	// plugins are required to support the --description argument.
 	// The Purpose attribute will be printed (if defined), allowing
 	// plugins to provide a sensible line of text for 'juju help plugins'.
-	f.BoolVar(&c.showDescription, "description", false, "")
+	f.BoolVar(&c.showDescription, "description", false, "Show short description of plugin, if any")
```

Comment:
Why add the usage string here given it's not previously been defined.

Reply:
Because otherwise you get an awkward blank line in the help output. If we're going to define a flag, we should describe it.

Reply:
(FWIW I don't really see why the flag is exported for all commands, but if it's going to be there, it should have some docs)